daft/datatype.py

```diff
@@ -431,7 +431,8 @@ def from_arrow_type(cls, arrow_type: pa.lib.DataType) -> DataType:
         elif isinstance(arrow_type, pa.BaseExtensionType):
             name = arrow_type.extension_name
 
-            if (get_context().runner_config.name == "ray") and (
+            runner_config = get_context().runner_config
+            if (runner_config and runner_config.name == "ray") and (
```

Comment:
Pretty unhappy about this, but the alternative is to call into `get_or_create_runner` which feels a little sus inside of datatypes? Maybe that's ok though.

Reply:
I guess its valid to assume that everything required to determine the runner, i.e. any env vars, set_runners, or ray_inits, have happened before this call.

Reply:
Yeah, changed.